source/UIAHandler/__init__.py

```diff
@@ -106,7 +106,8 @@
 textChangeUIAClassNames = (
 	"_WwG",  # Microsoft Word
 	"TermControl",
-	"TermControl2"
+	"TermControl2",
+	"WPFTermControl"

```

Comment:
For smaller diffs in future
```suggestion
	"WPFTermControl",
```